DasMulli.SimpleGitVersion/build/DasMulli.SimpleGitVersion.targets

```diff
@@ -96,7 +98,7 @@
  
     ========================================================================================================================
     -->
-  <Target Name="CreateGitVersionSuffix" BeforeTargets="PrepareForBuild;GenerateNuSpec" Condition=" '$(DisableGitVersionSuffix)' != 'true' ">
+  <Target Name="CreateGitVersionSuffix" BeforeTargets="_GenerateRestoreProjectSpec;CollectPackageReferences;PrepareForBuild" Condition=" '$(DisableGitVersionSuffix)' != 'true' ">
```

Comment:
This still allows "old" NuGet versions with some additional hacks to enforce nupkg targets participate in restore spec generation.